plugins/modules/find_perl_module.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/python
+
+# Copyright (c) 2023, Jonathan Kamens <jik@kamens.us>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# pylint: disable=missing-module-docstring
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: find_perl_module
+
+short_description: Determine from where to install Perl modules
+
+requirements:
+    - C(apt-file) executable in search path on systems that use the apt package
+      manager
+    - C(dnf) or C(yum) executable in search path on systems that use the
+      dnf/yum package manager
+    - C(cpanm) executable in search path if you want to be able to search for
+      packages using cpanm
+
+version_added: "1.0.0"
```

Comment:
```suggestion
version_added: "6.6.0"
```
That's the next release coming up.